bus/routes.go

```diff
@@ -181,6 +181,26 @@ func (b *Bus) consensusNetworkHandler(jc jape.Context) {
 	})
 }
 
+func (b *Bus) postSystemSQLite3BackupHandler(jc jape.Context) {
+	var req api.BackupRequest
+	if jc.Decode(&req) != nil {
+		return
+	}
+	switch req.Database {
+	case "main", "metrics":
+	default:
+		jc.Error(api.ErrInvalidDatabase, http.StatusBadRequest)
+		return
+	}
+	err := b.store.Backup(jc.Request.Context(), req.Database, req.Path)
+	if errors.Is(err, api.ErrBackupNotSupported) {
+		jc.Error(err, http.StatusNotFound)
```

Comment:
What's with the 404? 400 or 422 more appropriate

Reply:
Consistency with hostd. 400 and 422 imply that there is something the client can change to make the request or content valid but there isn't. If you don't use SQLite, this endpoint isn't available to you.